namer/serversets/src/test/scala/io/buoyant/namer/serversets/ServersetNamerTest.scala

```diff
@@ -1,16 +1,21 @@
 package io.buoyant.namer.serversets
 
 import com.twitter.finagle._
+import com.twitter.finagle.serverset2.BouyantZkResolver
 import com.twitter.util.{Activity, Var}
 import io.buoyant.namer.NamerTestUtil
 import java.net.{InetAddress, InetSocketAddress}
+import org.mockito.Matchers.{anyString, any}
+import org.mockito.Mockito.{verify, when, never}
 import org.scalatest.FunSuite
+import org.scalatestplus.mockito.MockitoSugar
 
-class ServersetNamerTest extends FunSuite with NamerTestUtil {
+class ServersetNamerTest extends FunSuite with NamerTestUtil with MockitoSugar {
```

Comment:
Mockito is pretty cool but I would prefer not to take on an additional testing framework at this time for the sake of consistency with other tests in the project.

You should be able to accomplish the same thing just as easily by overriding the `resolveServerset` similarly to how we already override the `resolve` method in this test.

Reply:
Thank you for the feedback @adleong.  I will make the changes you've requested.